Controller/Checkout/Redirect.php

```diff
@@ -103,7 +127,14 @@ public function execute(): void
         } catch (\Exception $e) {
             $this->_getCheckoutSession()->restoreQuote();
             $this->messageManager->addExceptionMessage($e, __('Something went wrong, please try again later'));
-            $this->messageManager->addExceptionMessage($e, $e->getMessage());
+            $this->logger->error(
```

Comment:
btw, should we store the sync error in the order object as failed? or is it some somewhere else?